denizen_scripts/survival/repo_link/temporary/world_event.dsc

```diff
@@ -0,0 +1,173 @@
+world_event_config:
+  type: data
+  materials:
+    iron_ingot: 1
+    iron_block: 9
+  server_amount_needed: 250000
+  personal_amount_needed: 15000
+
+currency_display_item:
+  type: item
+  debug: false
+  material: name_tag
+  display name: <&a>
+  mechanisms:
+    custom_model_data: 3
+
+world_event_bar_0:
+  type: item
+  debug: false
+  material: beetroot
+  display name: <&a>0%
+  mechanisms:
+    custom_model_data: 3
+
+world_event_bar_1:
+  type: item
+  debug: false
+  material: beetroot
+  display name: <&a>20%
+  mechanisms:
+    custom_model_data: 4
+
+world_event_bar_2:
+  type: item
+  debug: false
+  material: beetroot
+  display name: <&a>40%
+  mechanisms:
+    custom_model_data: 5
+
+world_event_bar_3:
+  type: item
+  debug: false
+  material: beetroot
+  display name: <&a>60%
+  mechanisms:
+    custom_model_data: 6
+
+world_event_bar_4:
+  type: item
+  debug: false
+  material: beetroot
+  display name: <&a>80%
+  mechanisms:
+    custom_model_data: 7
+
+world_event_bar_5:
+  type: item
+  debug: false
+  material: beetroot
+  display name: <&a>100%
+  mechanisms:
+    custom_model_data: 8
+
+world_event_progress_command:
+  type: command
+  name: open_world_event_window
+  permission: xeane.fuck_assignment_scripts_permission
+  script:
+    - run world_event_progress_inventory_open
```

Comment:
we'll need to notate #193 that the `/open_world_event_window` command requires the `xeane.fuck_assignment_scripts_permission` permission

Reply:
... actually reading the permission a second glance, if this is intended to be ran using command scripts you should ideally *just use an assignment script*
```suggestion
world_event_progress_assignment:
  type: assignment
  debug: false
  actions:
    on assignment:
      - trigger name:click state:true
    on click:
      - run world_event_progress_inventory_open
```

Reply:
Did you even read the permission name? Never!!!!